tests/kuttl/tests/dataplane-deploy-global-service-test/01-dataplane-deploy.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: custom-global-service
```

Comment:
this service isn't actually added anywhere, so the new code isn't tested. Did you mean to add this to servicesOverride on the Deployment?